src/main.rs

```diff
@@ -128,11 +133,22 @@ fn run() -> Result<()> {
 		return Ok(());
 	}
 
+	let log_level = match matches.opt_str("l").as_ref().map(String::as_ref) {
+		Some("0") => LogLevelFilter::Off,
+		Some("1") => LogLevelFilter::Error,
+		Some("2") => LogLevelFilter::Info,
+		Some("3") => LogLevelFilter::Debug,
+		_ => LogLevelFilter::Info,
+	};
+
+	TermLogger::init(log_level, Config::default()).unwrap();
+
 	#[cfg(unix)]
 	daemonize(&matches)?;
+	//attribute inside the if clause, because they are not yet allowed on `if` expressions
```

Comment:
I think this is a leftover from one of your earlier PR, can you remove this comment?